### ISSUE TITLE
Barcode scanning problems

### ISSUE THREAD

**Issue by lizheard1** [PlasticPatrol/PlasticPatrolWeb#185]
1. Incorrect product identified from barcode - e.g. lucozade bottle below
![Uploading WhatsApp Image 2020-09-21 at 10.49.08.jpeg…]()

2. Unable to find product by scanning the barcode 'lookup error', e.g. carlsberg can below
![Uploading WhatsApp Image 2020-09-21 at 10.49.09.jpeg…]()

**Comment by neilfulwiler** [comment]
yeah not sure we can really do anything abut this though, some barcodes just won't be in the system, and some will be wrong I guess. I don't really see how we can get around it though

**Comment by neilfulwiler** [comment]
I think we need some metrics on how often the suggestion from the barcode is used successfully. I've deployed ean lookup so that should cover some things taht were missed before

**Comment by neilfulwiler** [comment]
Also I found the icon screen in the depths of xcode that shows me what cordova actually builds from the config.xml file, so I should be able to attach the right sizes instead of using a single 1024x1024 icon for everything (which I'm not sure would pass review anyway) https://github.com/PlasticPatrol/PlasticPatrolWeb/pull/212/files

**Comment by sebastianovide** [comment]
@lizheard1 , is still a problem ?
